protected-api-access-protected-api/app.js

```diff
@@ -103,15 +105,10 @@ app.get('/me', validateJwt, (req, res) => {
       headers: { Authorization: `Bearer ${response.accessToken}` }
     }
 
-    // Collect the output from the HTTP GET request and send it back to the requesting API
-    const callback = function (graphResponse) {
-      graphResponse.on('data', function (chunk) {
-        res.send(chunk)
-      })
-    }
-
-    // Make the request against the Graph API
-    https.request('https://graph.microsoft.com/v1.0/me', options, callback).end()
+    // Make a request to the Graph /me endpoint and send the output to the requestor
+    https.request('https://graph.microsoft.com/v1.0/me', options, function (graph) {
+     graph.on('data', function (chunk) { res.send(chunk) })
+    }).end()
```

Comment:
I see you set `res.typoe('json')` outside of this, if you need to use it, I'd suggest putting it in like the following instead.

```suggestion
    // Make a request to the Graph /me endpoint and send the output to the requestor
    https.request('https://graph.microsoft.com/v1.0/me', options, function (graph) {
     graph.on('data', function (chunk) { 
       res.type('json')
       res.send(chunk)  // <-- but can instead you just use res.json(chunk) or res.json(JSON.parse(chunk)) and skip the res.type('json') call?
     })
    }).end()
```

Reply:
ok if I omit res.type('json')
res.json(chunk) - returns goofy looking things
but
res.json(JSON.parse(chunk)) - works correctly, so I can run with that

Reply:
thx!